src/os/bluestore/BlueFS.cc

```diff
@@ -929,6 +1164,7 @@ void BlueFS::_compact_log()
   bufferlist bl;
```

Comment:
I think we need to change op_jump_seq to specify a seq *and* a file offset... maybe just rename it jump?  then we can jump to the next 1MB-aligned offset, where the log will continue?